website/source/partials/helper/communicator/_SSH-not-required.html.md

```diff
@@ -42,12 +41,10 @@
     
 -   `ssh_agent_auth` (bool) - If true, the local SSH agent will be used to authenticate connections to
     the source instance. No temporary keypair will be created, and the
-    values of `ssh_password` and `ssh_private_key_file` will be ignored. To
-    use this option with a key pair already configured in the source AMI,
-    leave the `ssh_keypair_name` blank. To associate an existing key pair in
-    AWS with the source instance, set the `ssh_keypair_name` field to the
-    name of the key pair. The environment variable `SSH_AUTH_SOCK` must be
```

Comment:
Question: should I make a guide out of this ?

Reply:
Do you think the way it is right now is not complete? I would make it  a guide if it's incomplete and if it's something too complex to add here 🤔

Reply:
Yeah I agree with @sylviamoss here. I remember this issue https://github.com/hashicorp/packer/pull/8523 where this doc was added and there have not been many since.

Reply:
I think a guide would be useful to users, but isn't part of the scope of this PR.